File: web-app/src/DTOs/SignUpDTO.ts
export class SignUpDTO {
  public DisplayName: string;
  public FirstName: string;
  public LastName: string;
  public Email: string;
  public Password: string;
  public PasswordConfirm: string;

  constructor() {
    this.DisplayName = '';
    this.FirstName = '';
    this.LastName = '';
    this.Email = '';
    this.Password = '';
    this.PasswordConfirm = '';
  }
}

File: DavChatBot.Services/DTOs/SignUpDTO.cs
namespace DatChatBot.Services.DTOs
{
    public class SignUpDTO
    {
        public string DisplayName { get; set; } = string.Empty;
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
        public string PasswordConfirm { get; set; } = string.Empty;
    }

    public class SignUpDTOResponse
    {
        public bool Success { get; set; }
        public List<string> Errors { get; set; }

        public SignUpDTOResponse()
        {
            Errors = new List<string>();
        }
    }
}

